### ISSUE TITLE
Image preview block editing image link

### ISSUE THREAD

**Issue by skiokko** [madskristensen/CssTools#12]
The images preview on HTML files is extremely frustrating, I can't edit no more the image link due the preview go over the last part of the link. Please add the option to disable image preview or make it always small (not actual image size) and not over the link. Thank you!

**Comment by rolek** [comment]
Agreed, and sometimes it is huge, occupies the full editor.
An option to disable this feature would be nice.

**Comment by GlynP** [comment]
Yes please, me too. This is very frustrating. Too big. In the way of my code. An 'off switch' or containing the preview within its own windows would be good. The ability to decide between actual and a small preview size would also be good.
The HUGE 'No image available' for fonts that ARE in the folder displayed in the preview image is annoying too, for items like this:
src: url('../../Content/webfonts/copperplate/my-copperplate-font.eot');

**Comment by daiplusplus** [comment]
Image preview is handy, and I like how it also works with `data:` URIs, however it doesn't work for SVG files (both SVG file paths using `url()` or `data:`). Sometimes (I can't reproduce it) the (broken) SVG preview pop-up will fill 90% of my Visual Studio screen-space with an ugly upscaled "No image available" placeholder (I think this happens is when the `data:` URI is longer than a kilobyte or so).
